webdriver/tests/back/back.py

```diff
@@ -88,7 +87,7 @@ def test_fragments(session, url):
     assert session.url == test_pages[0]
 
 
-def test_history_pushstate(session, url):
+def test_history_pushstate(session, url, inline):
```

Comment:
Same here for `url`. Maybe scan through the changes and also correct others? Looks like a couple more exist.